public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
- [ ] (Optional) Kindly delete this file. It is of no use to your calculator application and deleting it would ensure that you follow _**YAGNI (You Ain't Gonna Need It)**_ principle.